openstack/nova/templates/hypervisors/hypervisors-vmware-deployment-vct.yaml

```diff
@@ -172,27 +118,75 @@ template: |
           - name: metrics
             containerPort: 9102
           volumeMounts:
-            - name: nova-etc
-              mountPath: /etc/statsd/statsd-exporter.yaml
-              subPath: statsd-exporter.yaml
-              readOnly: true
+            - name: statsd
+              mountPath: /etc/statsd
         volumes:
-        - name: etcnova
-          emptyDir: {}
         - name: nova-etc
-          configMap:
-            name: nova-etc
+          projected:
+            defaultMode: 420
+            sources:
+            - configMap:
+                items:
+                - key: nova.conf
+                  path: nova.conf
+                - key: policy.json
+                  path: policy.json
+                - key: logging.ini
+                  path: logging.ini
+                - key: rootwrap.conf
+                  path: rootwrap.conf
+                - key: compute.filters
+                  path: rootwrap.d/compute.filters
+                name: nova-etc
+            - configMap:
+                items:
+                - key: nova-compute.conf
+                  path: nova-compute.conf
+                - key: hostgroup-reservations.json
+                  path: hostgroup-reservations.json
+                name: nova-compute-vmware-{= name =}
+        - name: sudoers
+          projected:
+            defaultMode: 420
+            sources:
+            - configMap:
+                items:
+                - key: sudoers
+                  path: sudoers
         - name: hypervisor-config
           configMap:
             name: nova-compute-vmware-{= name =}
        {{- if .Values.neutron_dvs_agent_enabled }}
-        - name: etcneutron
-          emptyDir: {}
         - name: neutron-etc
-          configMap:
-            name: neutron-etc
-        - name: ml2-conf-vmware
-          configMap:
-            name: neutron-ml2-vmware-{= name =}
+          projected:
+            defaultMode: 420
+            sources:
+            - configMap:
+                items:
+                - key: neutron.conf
+                  path: neutron.conf
+                - key: api-paste.ini
+                  path: api-paste.ini
+                - key: policy.json
+                  path: neutron-policy.json
```

Comment:
Needs to be switched. See https://github.com/sapcc/helm-charts/blob/2ee3d7ef283269da28ed769fc20d8d503548ffa5/openstack/neutron/templates/vct-nsxv3-agent-deployment.yaml#L110-L111